src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+  type Props = {
+    movie: Movie;
+  };
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <>
+    <div className="card-image">
```

Comment:
There is one parent element in this component so the fragment is not strictly necessary2